src/packages/cli/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -306,13 +310,21 @@ async function setupScenario(kind: string, input: Input, scenario: Scenario) {
 
   await ctx.fs.dirAsync('.')
 
-  const dbClient = await input.database.connect(ctx)
-
-  state.db = dbClient
-
-  const databaseUpSQL = input.database.up?.(ctx) ?? ''
-  const upSQL = databaseUpSQL + scenario.up
-  await input.database.send(dbClient, upSQL)
+  if (input.database.name === 'sqlserver') {
```

Comment:
This `if` statement is also temporary. Once I can work on the test harness changes that were discussed, this will go inside mssql tests, rather than being in this common utility